examples/Consumer/Program.cs

```diff
@@ -40,7 +40,7 @@ public static void Run_Consume(string brokerList, List<string> topics, Cancellat
             {
                 BootstrapServers = brokerList,
                 GroupId = "csharp-consumer",
-                EnableAutoCommit = false,
+                EnableAutoOffsetStore = false,
```

Comment:
@mhowlett Referred https://github.com/edenhill/librdkafka/blob/master/INTRODUCTION.md#at-least-once-processing which mentions this to be false and AutoCommit to be enabled for `StoreOffset` to work. 

Please let me know if this is not right.

Reply:
this is right. i'd probably also explicitly configure `EnableAutoCommit` to `true` for clarity, even though it's the default.

Reply:
Thanks for confirming. Also enabled `EnableAutoCommit` in the config.